e2e/cypress/integration/enterprise/cloud/billing/subscriptions_free_trial_spec.js

```diff
@@ -0,0 +1,245 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @cloud_only @cloud_trial
+// Skip:  @headless @electron // run on Chrome (headed) only
+
+describe('System Console - Subscriptions section', () => {
+    before(() => {
+        // * Check if server has license for Cloud
+        cy.apiRequireLicenseForFeature('Cloud');
+
+        // # Visit Subscription page
+        cy.visit('/admin_console/billing/subscription');
+
+        // * Check for Subscription header
+        cy.get('.admin-console__header').should('be.visible').and('have.text', 'Subscriptions');
+    });
+
+    beforeEach(() => {
+        // # Click on close button of subscribe window if exist
+        cy.get('body').then(($body) => {
+            if ($body.find('#closeIcon').length > 0) {
+                cy.get('#closeIcon').parent().click();
+            }
+        });
+    });
+
+    it('MM-T4118 Visibility of Trial tag on Subscription screen', () => {
+        // * Check for visibility of Trial tag
+        cy.contains('span', 'Trial', {timeout: 10000}).should('be.visible');
+    });
+
+    it('MM-T4119 User Count on Subscription screen', () => {
+        // * Check for User count
+        cy.request('/api/v4/analytics/old?name=standard&team_id=').then((response) => {
+            cy.get('.PlanDetails__userCount > span').invoke('text').then((text) => {
+                const userCount = response.body.find((obj) => obj.name === 'unique_user_count');
+                expect(text).to.contain(userCount.value);
+            });
+        });
+    });
+
+    it('MM-T4126 "Contact Sales" navigation from Subscription screen', () => {
+        // * Check for Contact Sales navigation
+        cy.apiGetMe().then(({user}) => {
+            const adminUser = user;
+            const email = encodeURIComponent(adminUser.email);
+            const name = encodeURIComponent(`${adminUser.first_name} ${adminUser.last_name}`);
```

Comment:
This is good as is. However, it's repeated in several places. I still think this should be done in before hook as previously suggested.